azure-pipelines.yml

```diff
@@ -116,6 +116,87 @@ stages:
           flake8 --select F401 great_expectations tests
         name: UnusedImportChecker
 
+  - stage: docker_tests
+    dependsOn: []
```

Comment:
This should depend on scope check and other prior stages just to make sure we're not running the full suite on trivial changes (like README or docs updates). Same with condition.